A3A/addons/core/functions/CREATE/fn_createAttackVehicle.sqf

```diff
@@ -38,7 +38,7 @@ private _crewGroup = [_side, _vehicle, nil, _isAttackHeli] call A3A_fnc_createVe
 
 private _cargoGroup = grpNull;
 private _expectedCargo = ([_vehicleType, true] call BIS_fnc_crewCount) - ([_vehicleType, false] call BIS_fnc_crewCount);
-if (_expectedCargo >= 2 and !_isAttackHeli) then
+if (_expectedCargo >= 2) then
```

Comment:
I assume you want attack helis to carry cargo?

Reply:
Yes. Seems kinda silly to that something like Mi-48 Kajman will always do only heli attack run, so now it will land further then regular QRF during combat landing, or fastrope. During both it would be able to use it's weaponry.